packages/bruno-lang/v2/src/jsonToBru.js

```diff
@@ -4,6 +4,10 @@ const { indentString } = require('../../v1/src/utils');
 
 const enabled = (items = []) => items.filter((item) => item.enabled);
 const disabled = (items = []) => items.filter((item) => !item.enabled);
+const quoteKey = (key) => {
+  const quotableChars = [':', '"', '{', '}', ' '];
```

Comment:
The characters
```
:  {  } "
```
are important for bru syntax, and so they should be escaped by default. The question of space is open for discussion, as it may be solved in a different way (#2898) without forcing changes onto users' bru files.